source/includes/_06a-one-time-callback-docfetch.html.md.erb

```diff
@@ -0,0 +1,758 @@
+                'type': 'PROVIDER_CONNECTION',
+                'message': f'Failed to contact data provider: {str(e)}'
+            }],
+            'warnings': []
+        })
+    except ProviderError as e:
+        return jsonify({
+            'errors': [{
+                'type': 'PROVIDER_MESSAGE',
+                'message': f'Provider error: {str(e)}'
+            }],
+            'warnings': []
+        })
+```
+
+>This endpoint should return JSON in the following format:
+
+```json
+<%= pretty_json("responses/finish_check_callback_document_fetch", "json") %>
+```
+
+### Response fields
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>check_output</code></td>
+      <td>Object structured per <code>check_output</code> below</td>
+      <td>If check did not error</td>
+      <td>
+        Indicates the result data for this check. See the
+        <code>check_output</code> section for the full structure of
+        this check.
+      </td>
+    </tr>
+    <tr>
+      <td><code>errors</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-errors">Errors</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of errors that occurred while running this check. If this is
+        provided and it's not empty, the result of this check will be
+        considered to be errored by PassFort, regardless of the other contents
+        of this response.
+      </td>
+    </tr>
+    <tr>
+      <td><code>warnings</code></td>
+      <td>
+        An array containing one or more warnings
+      </td>
+      <td>No</td>
+      <td>
+        Essentially the same as <code>errors</code>, except the
+        <code>sub_type</code> and <code>data</code> fields are not present, and
+        providing warnings will not cause PassFort to consider the check
+        as errored.
+      </td>
+    </tr>
+    <tr>
+      <td><code>provider_data</code></td>
+      <td>
+        Any valid JSON value
+      </td>
+      <td>No</td>
+      <td>
+        This should be the structured JSON data returned by the data provider,
+        or a conversion of the data provider's response, to allow
+        PassFort to investigate issues that may arise with your integration.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+#### `check_output`
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>entity_type</code></td>
+      <td>The value <code>"INDIVIDUAL"</code></td>
+      <td>Yes</td>
+      <td>
+        The only supported entity type for Document Verification Checks
+        is individuals.
+      </td>
+    </tr>
+    <tr>
+      <td><code>documents</code></td>
+      <td>
+        An array containing the documents, structured as it is
+        in <a href="#passfort-data-structure">the PassFort API</a>.
```

Comment:
This link doesn't work when I serve the docs locally.

Reply:
Ah, I think the section head was updated in a previous commit and no longer works. It's actually broken in several places...